File: geek-university-algoritmo-e-logica-de-programacao/c/exercicio_8.c
#include <stdio.h>

int main() {
	int num1, num2, soma;

	printf("Informe o primeiro numero: ");
	scanf("%d", &num1);
	printf("Informe o segundo numero: ");
	scanf("%d", &num2);

	soma = (num1 + num2);

	printf("A soma de %d com %d eh igual a ", num1, num2, soma);
}

File: geek-university-algoritmo-e-logica-de-programacao/java/exercicio_8.java
import java.util.Scanner;

public class exercicio_8 {

	public static void main(String[] args) {
		int num1, num2, soma;
		Scanner teclado = new Scanner(System.in);

		System.out.print("Informe o primeiro numero: ");
		num1 = teclado.nextInt();
		
		System.out.print("Informe o segundo numero: ");
		num2 = teclado.nextInt();

		soma = (num1 + num2);

		System.out.println("A soma eh " + soma);
		
		teclado.close();
	}

}
